diff -Naur mikutter.org/core/mui/gtk_extension.rb mikutter/core/mui/gtk_extension.rb
--- mikutter.org/core/mui/gtk_extension.rb	2010-07-04 02:22:26.797614682 +0900
+++ mikutter/core/mui/gtk_extension.rb	2010-07-04 02:23:28.568739787 +0900
@@ -136,4 +136,4 @@
     shellExecuteA = Win32API.new('shell32.dll','ShellExecuteA',%w(p p p p p i),'i')
     shellExecuteA.call(0, 'open', url, 0, 0, 1)
   else
-    system("/etc/alternatives/x-www-browser #{url} &") || system("firefox #{url} &") end end
+    system("xdg-open #{url} &") || system("firefox #{url} &") end end
diff -Naur mikutter.org/mikutter.rb mikutter/mikutter.rb
--- mikutter.org/mikutter.rb	2010-07-04 02:22:26.798614595 +0900
+++ mikutter/mikutter.rb	2010-07-04 02:22:57.077740526 +0900
@@ -1,7 +1,7 @@
 #! /usr/bin/ruby
 # -*- coding: utf-8 -*-
 
-Dir.chdir(File.join(File.dirname($0), 'core'))
+Dir.chdir('/usr/share/mikutter')
 
 require 'utils'
 miquire :core, 'environment'
